decidim-proposals/app/views/decidim/proposals/proposals/_endorsement_button.html.erb

```diff
@@ -0,0 +1,25 @@
+<%
+# @params (mandatory): proposal, from_proposals_list
+# @params (optional) : user_group, btn_label
+# avoid "undefined local variable" error
+user_group= local_assigns[:user_group]
```

Comment:
Missing space between variable name and `=` sign (next lines have the same problem)

Reply:
Oh god! when will linters realize that concatenating the equal sign just after the assigned variable makes code much more readable than putting a space between! I'm giving up! :sob: